website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,107 @@
+## How typography is referenced
+
+The construction of typographic styles varies heavily depending on context and tooling. This overview will clarify how HDS typography standards are implemented in different tools.
+
+In Figma, reusable text styles are called “Text Styles” and are categorized and stored in the “text panel” to be used in text-based elements. Text styles directly reference font family, weight, line height, size, and many other text-based properties within a single style and are manually maintained. Some Figma styles include:
+
+- `Display/500/Bold`
+- `Body/300/Regular`
+- `Code/300/Regular`
+
+In code, these styles are constructed differently, using “Tokens.” Tokens are generated from an agnostic JSON file and differ from Figma “Text Styles” because a single token does not reference multiple font properties. Instead, each Token defines the different properties of a single text style. This means that constructing the equivalence of Figma’s Text Style `Display/500/Bold` would utilize the following tokens:
+
+- `font-family: var(--token-typography-display-500-font-family);`
+- `font-size:var(--token-typography-display-500-font-size);`
+- `line-height:var(--token-typography-display-500-line-height);`
+- `font-weight:var(--token-typography-font-weight-bold);`
+
+Rather than calling upon each of these tokens in code every time text is styled, the [Text Component](/components/text) was created to replicate how Figma styles group text-based properties.
+
+## Font families
+
+Helios uses the default system fonts available on each OS. These system fonts are stable and support internationalization, code, tabular data, data visualization, etc. Additionally, they improve performance by reducing the number of resources to be loaded, which is important for large applications like those at HashiCorp.
+
+While text styling varies slightly across operating systems, users will have a consistent and expected experience within each OS.
+
+### Sans-serif
+
+![sans-serif on mac](/assets/foundations/typography/sans-serif-mac.png =660x*)
```

Comment:
```suggestion
![The sans-serif font on Mac is SF Pro Display for 20px and above and SF Pro Text for 19px and below.](/assets/foundations/typography/sans-serif-mac.png =660x*)
```

Reply:
Suggestion: could be out of scope for this PR, but we could rethink these screenshots to move some of the text content out of the image and into the HTML of the page. This would  benefit users because long alt text is not recommended from an a11y perspective and the text would be searchable.

Reply:
an alternate design could be like this (ignore my hacky Figma usage lol)
![Screenshot 2024-08-27 at 5 27 23 PM](https://github.com/user-attachments/assets/f5752706-fe22-4bf7-a657-fdcc8efc7ed2)

Reply:
I agree that would would be better to include less of the text within the images such as the labels for the particular operating system.

Reply:
This is a good idea!